lib/ember_cli/shell.rb

```diff
@@ -37,6 +37,7 @@ def install
         exec "#{paths.bundler} install"
       end
 
+      exec "#{paths.ember} version || rm -rf #{paths.npm_deps} #{paths.bower_deps}"
```

Comment:
Hmm, looks like `exec` will raise an exception if the passed command returns non-zero. I suppose I could just use `Kernel#system`, but that doesn't provide the same guarantees or logging as `exec`. WDYT?